packages/firebase_mlkit_language/lib/src/language_identifier.dart

```diff
@@ -0,0 +1,76 @@
+part of firebase_mllanguage;
+
+/// Used for finding [LanguageLabel]s in supplied text.
+///
+///
+/// A language identifier is created via
+/// `languageIdentifier([LanguageIdentifierOptions options])` in [FirebaseLanguage]:
+///
+/// ```dart
+///
+/// final LanguageIdentifier languageIdentifier =
+///     FirebaseLanguage.instance.languageIdentifier(options);
+///
+/// final List<LanguageLabel> labels = await languageIdentifier.processText("Sample Text");
+/// ```
+
+class LanguageIdentifier {
```

Comment:
I would call this FirebaseLanguageIdentification for consistency with iOS/Android.